content/en/contribute/code/workflow.md

```diff
@@ -69,6 +69,12 @@ All features and bug fixes must have at least one unit test. All features must h
 
 Those are minimums. Our ultimate goal is to have fully-automated release testing, allowing for fast, confident delivery of completed code. If your work would be included in a release/regression test, create an e2e test for it. QA engineers will use that as a template of sorts to create additional automated tests at their discretion.
 
+### Release Testing
+The release testing suite covers a range of tests to check for any regressions, performance tests, and scalability tests. We have a release [testing repository](https://github.com/medic/cht-release-testing) with major workflows. Most issues will have a step-by-step description or a link to some details for execution. At release tesing time, create a [project board](https://github.com/medic/cht-release-testing#generating-a-project-board-for-testing-a-release) with all issues labelled 'release'.
+Start testing following instructions on the ticket. If the test passes move it to `pass`. If it fails move it to `fail`, raise an issue and notify the team. A decision will be made whether to halt the release (blocker) or schedule the fix for the next release. [Here](https://github.com/medic/cht-release-testing/projects/101) is a typical release test project with results in the [comment section](https://github.com/medic/cht-release-testing/issues/190#issuecomment-1032646259) on individual tickets.
+As part of release testing, there is a [scalablity test suite ](https://github.com/medic/cht-core/tree/master/tests/scalability) to test the scalability of CouchDB so we can check what our limits are and ensure we don't regress on performance. Initially it only tests replication but should be extended to test other APIs. The JMeter tests are run on an [AWS](https://github.com/medic/cht-core/tree/master/tests/scalability#running-tests-on-aws) install and a report is saved in an S3 bucket. We then compare results from previous runs and save a JTL file containing the results of the test run in `cht-core/tests/scalability/previous_results`.
```

Comment:
The scalability test hits API so it tests basically everything to do with replication from API on down to the DB.

```suggestion
As part of release testing, there is a [scalability test suite](https://github.com/medic/cht-core/tree/master/tests/scalability) to test the scalability of the CHT server so we can check what our limits are and ensure we don't regress on performance. Initially it only tests replication but should be extended to test other APIs. The JMeter tests are run on an [AWS](https://github.com/medic/cht-core/tree/master/tests/scalability#running-tests-on-aws) install and a report is saved in an S3 bucket. We then compare results from previous runs and save a JTL file containing the results of the test run in the [cht-core repository](https://github.com/medic/cht-core/tree/master/tests/scalability/previous_results).
```